pages/in/widget/iscn-ar/index.vue

```diff
@@ -623,12 +575,13 @@ export default {
       this.isUsingKeplr = true;
       // Direct register ISCN flow
       if (this.actualSendAmount === 0 || this.actualSendAmount === '0') {
-        this.mainStatus = 'directRegisterISCN';
+        this.mainStatus = 'registerISCN';
         this.submitISCNTransfer();
         return;
       }
       // pay LIKE Pay
       this.mainStatus = 'LIKEPay';
+      [this.transactionFee] = [this.amounts[0]];
```

Comment:
Why not just `[this.transactionFee] = this.amounts`?

Reply:
okie